osu.Game/Updater/GameUpdateManager.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+
+namespace osu.Game.Updater
+{
+    /// <summary>
+    /// An update manager which only shows notifications after a game update completes.
+    /// </summary>
+    public partial class GameUpdateManager : UpdateManager
```

Comment:
These classes just keep on coming...

- Why does this exist at all?
- Why is `SquirrelUpdateManager` not a `GameUpdateManager`, given it updates the game?

This class needs to not exist, ideally.